packages/Webkul/Velocity/src/Routes/front-routes.php

```diff
@@ -80,6 +80,9 @@
     Route::delete('/cart/remove/{id}', [CartController::class, 'removeProductFromCart'])
         ->name('velocity.cart.remove.product');
 
+    Route::get('/cart/remove-all', [CartController::class, 'removeAllItems'])
```

Comment:
It should not be the GET method and it should be DELETE or POST with CSRF Token.

Reply:
Ok! change method type

Reply:
@devansh-webkul Ok! change method type